PYME/Acquire/Spooler.py

```diff
@@ -204,23 +205,18 @@ def doStartLog(self):
         self.dtStart = dt
         
         self.tStart = time.time()
-          
-        mdt = MetaDataHandler.NestedClassMDHandler()
-          
-        mdt.setEntry('StartTime', self.tStart)
 
-        #loop over all providers of metadata
+        # loop over all providers of start-metadata
         for mdgen in MetaDataHandler.provideStartMetadata:
-            mdgen(mdt)
-            
-        self.md.copyEntriesFrom(mdt)
```

Comment:
Generating a blank metadata handler, populating, and then copying to series metadata handler (rather than populating the series handler directly) is a critically important performance optimization, resulting in a single write rather than multiple writes to storage (at least for HDF and Queue targets - the Cluster spooler caches internally and is probably immune to the performance issues)

Reply:
How much do we care? Critically important for performance optimization of the 95% deprecated localization analysis enough not to speed up the fast version?
 
![image](https://user-images.githubusercontent.com/31105780/75175352-29230980-5700-11ea-9231-8eb7de15afa8.png)

Reply:
Not just queue spooling - spooling directly to disk (which is not deprecated) is also affected. 

Even if it were only the queue spooling, I'm not prepared to break that this badly at this point.  The alternative being 95% ready does not equate to the old-style being deprecated yet - there are a lot of people using it and I'm not prepared to arbitrarily break it for them. IE we care a lot

I suspect that any difference that you are seeing in metadata collection between the collect and copy vs direct collect is likely to be a profiling artifact, and in any case you need to show the comparative cost (which is unlikely to change). Even given profiling overhead, those figures seem a little on the high side though, so might be worth looking into further, but the `mdgen` cost should be the same for both new and old code.

Reply:
The mdgen cost is high because the current repo head features 3x clusterIO exist calls and one of my maps isn't cached. 

Do you have time for a quick skype @David-Baddeley?

Reply:
Using ipython to time things without profiler overhead ...

```python
>>> from PYME.IO import MetaDataHandler
>>> mdh = MetaDataHandler.load_json('/Users/david/srvtest/test1/david/2019_5_10/10_5_series_B.pcs/metadata.json')
>>> md2 = MetaDataHandler.NestedClassMDHandler()
>>> %timeit md2.copyEntriesFrom(mdh)
1000 loops, best of 3: 1.37 ms per loop
>>> %timeit md3 = MetaDataHandler.NestedClassMDHandler(mdh)
1000 loops, best of 3: 1.19 ms per loop
```

the metadata stuff is fast enough (~1ms) that we shouldn't care